### PR TITLE
using rsync over CIFS

### PR DESCRIPTION
**Pull request by sboesebeck**
as discussed, this simplifies the archive-clips.sh a lot. It is more or less a copy of the existing rsync-system.

### CODE REVIEW

File: run/cifs_archive/archive-clips.sh [hunk 1]
```diff
@@ -1,96 +1,16 @@
...
-rmdir --ignore-fail-on-non-empty "$CAM_MOUNT/TeslaCam/SavedClips"/* "$CAM_MOUNT/TeslaCam/SentryClips"/* || true
-
-log "Moved $NUM_FILES_MOVED file(s), failed to copy $NUM_FILES_FAILED, deleted $NUM_FILES_DELETED."
-
-if [ $NUM_FILES_MOVED -gt 0 ]
+if (( $num_files_moved > 0 ))
 then
-  /root/bin/send-push-message "TeslaUSB:" "Moved $NUM_FILES_MOVED dashcam file(s), failed to copy $NUM_FILES_FAILED, deleted $NUM_FILES_DELETED."
+  find $CAM_MOUNT/Teslacam/SavedClips/ $CAM_MOUNT/Teslacam/SentryClips/ -depth -type d -empty -exec rmdir "{}" \;
+  log "Successfully synced files through rsync."
```
Comment (marcone) [inline run/cifs_archive/archive-clips.sh]: would be nice if this message also included $num_files_moved

**Review by marcone — COMMENTED** [review]
When I tried this it moved the SentryClips folders to the archive dir, instead of to archivedir/SentryClips like the current code does.

File: run/cifs_archive/archive-clips.sh (hunk 1 above)
Comment (sboesebeck) [reply run/cifs_archive/archive-clips.sh]: added the number of files syned to the log message also. 
the paths should be fixed as well

**Comment by RichieB2B** [comment]
This fixes #294

**Pull request merged**